### ISSUE TITLE
FlipperOkhttpInterceptor making SSE not working

### ISSUE THREAD

**Issue by tmirun** [facebook/flipper#2495]
## 🐛 Bug Report

in the reactNativeFlipper.java, the following lines making EventSource (SSE) not working.

```javascript
....
NetworkFlipperPlugin networkFlipperPlugin = new NetworkFlipperPlugin();
      NetworkingModule.setCustomClientBuilder(
          new NetworkingModule.CustomClientBuilder() {
            @Override
            public void apply(OkHttpClient.Builder builder) {
              builder.addNetworkInterceptor(new FlipperOkhttpInterceptor(networkFlipperPlugin));
            }
          });
```

## Environment
Android, Flipper 0.93, react native 0.64.0

**Comment by tmirun** [comment]
you can follow this conversation:
https://github.com/facebook/react-native/issues/28835

**Comment by axwell** [comment]
any news about this ?

**Comment by tmirun** [comment]
@axwell in production mode SSE it is working, that `networkFlipperPlugin` is only in development mode, for you can intercept the network calls and show it in Flipper.

**Comment by mweststrate** [comment]
We don't really use the OkHttpClient interceptor internally, so if anybody wants to investigate if this could be wired better, PRs are welcome!

**Comment by KillianH** [comment]
I'll like to keep this issue open :) We still have the same problem on our side too.

**Comment by mweststrate** [comment]
You're interested in investigating it further? Otherwise there is little value in keeping issues open that no one is willing to invest in.

**Comment by forkball** [comment]
Yeah, ran into this issue at work today. Works fine on release versions of Android apps, but not debug. What does this specific snippet of code do? Is it required to use Flipper in normal debugging practices?

**Comment by forkball** [comment]
Still looking for some insight into this code snippet.

What does this specific snippet of code do? Is it required to use Flipper in normal debugging practices?

**Comment by fa2id** [comment]
Same here. @forkball  Did you find any solution?

**Comment by forkball** [comment]
> Same here. @forkball Did you find any solution?

I just commented the snippet in question out for the time being.

**Comment by yusinto** [comment]
We don't officially support expo. However, there is a known issue with react native android where Flipper intercepts sse streaming connections causing them to fail. See https://github.com/facebook/flipper/issues/2495. The symptoms described here seem to be pointing to that.

Production android builds don't use Flipper and so don't suffer from this issue.
